compiler/luci/import/src/GraphBuilderRegistry.cpp

```diff
@@ -116,7 +116,8 @@ GraphBuilderRegistry::GraphBuilderRegistry()
   CIRCLE_NODE(RESIZE_NEAREST_NEIGHBOR, CircleResizeNearestNeighborGraphBuilder);           // 97
   CIRCLE_NODE(REVERSE_SEQUENCE, CircleReverseSequenceGraphBuilder);                        // 112
   CIRCLE_NODE(REVERSE_V2, CircleReverseV2GraphBuilder);                                    // 105
-  CIRCLE_NODE(RMS_NORM, CircleRmsNormGraphBuilder);                                        // 255
+  CIRCLE_NODE(RMS_NORM, CircleRmsNormGraphBuilder);                                        // 250
```

Comment:
It is not related to this PR, but it is a simple comment change,
 so I modify it. RMS_NORM op 255 -> 250

Reply:
Plz split this to another PR.

Reply:
> Plz split this to another PR.

thanks, i updated

@seockho-kim 
I think it would be better to modify the op number of RMS_NORM.
please check this comment.

Reply:
> > Plz split this to another PR.
> 
> thanks, i updated
> 
> @seockho-kim I think it would be better to modify the op number of RMS_NORM. please check this comment.

Ok, I got it
Thanks :)